src/CosmosDB/tools/deploy.ps1

```diff
@@ -0,0 +1,63 @@
+﻿$extensionPath=Split-Path -parent $PSCommandPath
```

Comment:
can we restructure this as a PS command add-extension.ps1 and make it take parameters. then we can add the details on how to run this in the readme